ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -90,6 +90,12 @@ queryNoWith:
       (LIMIT limit=(INTEGER_VALUE | ALL))?
     ;
 
+printClause
+      : (FROM BEGINNING)?
+      ((INTERVAL | SAMPLE) number)?
+      (LIMIT limit=(INTEGER_VALUE | ALL))?
```

Comment:
Why do we need `ALL`? isn't ignoring `LIMIT` already doing this?

Reply:
Yeah... I was debating whether or not to add this or not and figured that keeping it consistent with the LIMIT clause for other queries was preferable but I'm willing to be convinced that in this use case we can just remove it. Thoughts?

```
queryNoWith:
      queryTerm
      (LIMIT limit=(INTEGER_VALUE | ALL))?
    ;
```

Reply:
+1 for consistency

Reply:
I haven't seen the LIMIT ALL in any DB except Pestro. Is it useful to have the LIMIT ALL in Ksql?

Reply:
Well, as @spena said we have it in presto and it comes to KSQL from presto grammar. I would remove it as grammar clean up since we haven't explicitly used it so far! We can add it again if we need it.

Reply:
The newest patch removes `ALL` from KSQL entirely.